.github/workflows/codeql-analysis.yml

```diff
@@ -53,7 +53,7 @@ jobs:
 
     # Manually build the java bytecode
     - name: Execute Gradle build
-      run: ./gradlew assembleLibraries
+      run: ./gradlew assembleLibrariesDebug
```

Comment:
ideally we still need to build both build types (because it is a security scan, it needs to cover everything) or at least `release` build type instead of `debug`, because `release` is what will be used by the customer later.